apps/builder/app/shared/commands-emitter.ts

```diff
@@ -57,7 +57,7 @@ const findCommandsMatchingHotkeys = (event: KeyboardEvent) => {
     pressedKeys.add("shift");
   }
   if (altKey) {
-    pressedKeys.add("altKey");
+    pressedKeys.add("alt");
```

Comment:
drive-by fix, I think we are not using alt yet, but it was inconsistent with others

Reply:
good catch